str/trtools/merge_str_prep.py

```diff
@@ -62,7 +54,7 @@ def main(
 
     input_vcf_dict = {}
 
-    for id in list(external_id_to_cpg_id.values()):
+    for id in list(internal_wgs_ids):
         input_vcf_dict[id] = os.path.join(input_dir, f'{id}_{caller}.vcf')
```

Comment:
```suggestion
    input_vcf_dict = {id: os.path.join(input_dir, f'{id}_{caller}.vcf') for id in internal_wgs_ids}
```

This dict creation can be tidied into a single statement